apiserver/facades/client/application/application.go

```diff
@@ -1396,13 +1388,9 @@ func (api *APIBase) addApplicationUnits(
 		}
 		attachStorage[i] = tag
 	}
-	oneApplication, err := api.backend.Application(args.ApplicationName)
-	if err != nil {
-		return nil, errors.Trace(err)
-	}
```

Comment:
Can't do this anymore, as we don't dual write applications during bootstrap. This also means we're no longer dual writing adding units 🎉